tests/test_api.py

```diff
@@ -46,7 +46,7 @@ def build_fn_clf(
     model.add(keras.layers.Dense(n_classes_))
     model.add(keras.layers.Activation("softmax"))
     model.compile(
-        optimizer="sgd", loss="categorical_crossentropy", metrics=["accuracy"]
+        optimizer="sgd", loss="sparse_categorical_crossentropy", metrics=["accuracy"]
```

Comment:
The reason that we need all of these changes is that previously SciKeras would look at `model_.loss` and see `categorical_crossentropy` and then apply `OneHotEncoder`. Now it won't do that unless you pass it as `loss=...` in `__init__`. It was a much smaller diff and easier to just rename `categorical_crossentropy` to `sparse_categorical_crossentropy` like I did here than to add the `loss=...` parameter to every, single, test.